honeycomb-core/src/cmap/dim2/components.rs

```diff
@@ -0,0 +1,87 @@
+// ------ IMPORTS
+
+use std::{
+    ops::Index,
+    sync::atomic::{AtomicBool, AtomicU32},
+};
+
+use crate::cmap::{DartIdentifier, NULL_DART_ID};
+
+// ------ CONTENT
+
+// --- beta functions storage
+
+pub struct BetaFunctionsAt<const N: usize>(Vec<[AtomicU32; N]>);
```

Comment:
>Not clear what the N parameter is.

It's the number of beta functions, including beta 0. I'll add documentation, and probably a check so that it is non-zero

>The name of the structure is quite strange also.

It was in reference to atomics, if we want to define variants that use transactional variables or something. I'll change it since we might go full transactional anyway due to issues in #201